users/faq-parts/usage.rst

```diff
@@ -269,13 +269,11 @@ How do I reset the GUI password?
 --------------------------------
 
 If you've forgotten / lost the GUI password, you can reset it using the
-:option:`--gui-password` (and possibly :option:`--gui-user`) options to the
-``syncthing generate`` subcommand.  This should be done while Syncthing is not
-running.
+:option:`syncthing cli config gui password` (and possibly :option:`syncthing cli config gui user`) subcommands.  
+
+1. ``syncthing cli config gui user set newUserName``
+2. ``syncthing cli config gui password set myNewPassword``
 
-1. Stop Syncthing: ``syncthing cli operations shutdown``
-2. ``syncthing generate --gui-password=myNewPassword --gui-user=newUserName``
```

Comment:
The "step 2" referenced below is now something different.  In fact, the whole following paragraph doesn't make much sense if you only document the CLI commands.